pkg/sources/adapter/awssnssource/handler/handler.go

```diff
@@ -100,7 +100,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		h.logger.Debug("Request body: ", string(body))
+		h.logger.Debugw("Received Notification request", zap.String("body", string(body)))
```

Comment:
I can only speak about `zap`'s **core** encoder implementations, `json` and `console`, but in both cases I observed that wrapping the request body that way effectively printed newline characters as literal `\n` and `\r`.